lms/static/sass/discussion/_discussion.scss

```diff
@@ -1066,12 +735,3 @@ body.discussion {
   }
 }
 
-// ====================
```

Comment:
Context: this section was moved to line 94 of _thread.scss, under the area labeled by the following comment: // thread - header content details.